cypress/e2e/spec.ts

```diff
@@ -102,27 +102,31 @@ describe('workspace-project App', () => {
 
     cy.intercept('GET', 'https://app.kendra.io/api/workflowCloud/listWorkflows', {
       fixture: 'listWorkflowsFlow.json'
-    });
+    }).as('listWorkflowsFlow.json');
 
     cy.intercept('GET', 'https://app.kendra.io/api', {
       fixture: 'flowList.json'
     }
     ).as('flowList.json');
 
     cy.visit('/workflowCloud/listWorkflows');
+    cy.contains('Submit').click();
     cy.contains('Made up flow A');
   });
 
-
+  /*
```

Comment:
Thanks Gio, now I've fixed it using more robust selectors and checks to make sure we wait for the right state.
The comment to disable it was introduced a long time ago in the Angular 16 upgrade branch, for the clarity of a small, readable and more simple diff, I deliberately didn't address it as part of the tidying and squashing but it obviously did need fixing, and happily, now it is.
I made the code easier to understand too, since I refactored it quite a bit it's easier to just read the new version for it that the diff:
https://github.com/kendraio/kendraio-app/blob/b429dc24f1326c742bdd246e2340bae1733df143/cypress/e2e/spec.ts#L117
@gsambrotta

Reply:
Awesome!